src/utils/shared_macros.py

```diff
@@ -4,46 +4,52 @@
 
 def COLOR(color, msg):
     """
-    @brief      Wraps a string to print with ANSI terminal colors
+    Wraps a string to print with ANSI terminal colors
 
-    @param      color  The color to use
-    @param      msg    The message to wrap
+    :param      color:  The color to use
+    :type       color:  str
+    :param      msg:    The message to wrap
+    :type       msg:    str
 
-    @return     New colored text
+    :returns:   New colored text
+    :rtype:     str
     """
     if color == "black":
-        return "\033[30m{}\033[0m".format(msg)
+        return f"\x1b[30m{msg}\x1b[0m"
```

Comment:
Why is this changed?

Reply:
This was automatically changed when I ran this through `flynt`. I tested the changes out and the colour is the same, I think the linter just changed the values from octal to hexadecimal.